tests/test_suites/ShortSystemTests/test_cs_failure_recovery_speed.sh

```diff
@@ -9,15 +9,25 @@ CHUNKSERVERS=4 \
 mkdir "${info[mount0]}/dir"
 saunafs setgoal 4 "${info[mount0]}/dir"
 head -c 1M /dev/urandom | tee "${info[mount0]}"/dir/file{0,1,2,3,4} >/dev/null
+if is_windows_system; then
+	sleep 2
+fi
 
 # Stop one of chunkservers which has one copy of each chunk and
 # immediately start overwriting all the chunks.
-kill -s SIGSTOP "$(saunafs_chunkserver_daemon 0 test 2>&1 | sed 's/.*pid: //')"
+if is_windows_system; then
+	saunafs_chunkserver_daemon 0 stop
+else
+	kill -s SIGSTOP "$(saunafs_chunkserver_daemon 0 test 2>&1 | sed 's/.*pid: //')"
+fi
```

Comment:
Well, there are changes in fact. The main idea behind this change is that under WSL1 looks like some signals are successfully processed or something like that because the saunafs masters and chunkservers "cannot find themselves". 

For instance, using two times the command "sfschunkserver -d" creates two processes rather than stopping the first one. The second chunkserver process ends up indefinately asking to connect to master.

The current status of the PR tries to tackle this issue with the function ```windows_server_aux``` from the saunafs.sh file. The idea is to directly find the process and send the signal with the ```kill``` command. It only implemented the reload, kill, stop and start commands, though start might not be strightforward. The case is that I could not get the test command running.

This test case, in its oriignal version uses ```saunafs_chunkserver_daemon 0 test``` which is not working right now, and just left the command in the much simpler stop command.